src/Symfony/Component/Security/Core/Authorization/AccessDecisionManager/HighestNotAbstainedVoterAccessDecisionManager.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Symfony\Component\Security\Core\Authorization\AccessDecisionManager;
+
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Authorization\Voter\VoterInterface;
+
+/**
+ * Grants access depending on the first not abstained voter decission.
```

Comment:
by the way, the class is called `Highest ...` and here we mention `first ...` Should we change `HighestNotAbstainedVoterAccessDecisionManager` by `FirstNotAbstainedVoterAccessDecisionManager` ?